### PR TITLE
fix(workflows): remove `tag=` prefix for action digest tags

### PR DESCRIPTION
**Pull request by eglitise**
## Proposed changes

Renovate appears to have forgotten how to parse GitHub Action digest tags with the `tag=` prefix. Given that tags without this prefix seem to be parsed just fine, let's see if removing the prefix fixes the problem.

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [X] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [X] I have read the [CONTRIBUTING](https://github.com/appium/appium/blob/master/CONTRIBUTING.md) doc
- [X] I have signed the CLA
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by KazuCocoa — APPROVED** [review]

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -77,7 +77,7 @@ jobs:
           node-version: ${{ matrix.node-version }}
           cache: 'npm'
       - name: Install dependencies
-        uses: bahmutov/npm-install@1a235c31658a322a3b024444759650ee6345c26d # tag=v1
+        uses: bahmutov/npm-install@1a235c31658a322a3b024444759650ee6345c26d # v1
```
Comment (KazuCocoa) [inline .github/workflows/build.yml:80]: or i guess we can use `@v1`  instead

Comment (eglitise) [reply .github/workflows/build.yml:80]: Then Dependabot would just create a PR to pin them (like #18866)

Comment (KazuCocoa) [reply .github/workflows/build.yml:80]: Maybe removing https://github.com/search?q=repo%3Aappium%2Fappium%20pinGitHubActionDigests&type=code would help...?
i haven't configured renovate well but maybe the helper does the work...?
https://docs.renovatebot.com/modules/manager/github-actions/

Comment (eglitise) [reply .github/workflows/build.yml:80]: This can of course be done, but is it not better to keep them pinned?

Comment (mykola-mokhnach) [reply .github/workflows/build.yml:80]: I personally get annoyed by these renovate messages. I am ok to keep actions manually pinned to a specific tag like we do in other projects where we don't use renovate

Comment (eglitise) [reply .github/workflows/build.yml:80]: Hmm, fair enough - updating only on major tag changes would still be fine. In that case I will close this PR and create a new one that removes the digests from all actions, and adjusts the Renovate config to not pin them.

**Pull request closed without merging**